popde/density_estimate.py

```diff
@@ -18,9 +18,39 @@ def __init__(self, data, backend='scipy', bandwidth=1., dim_names=None):
                points of the data define each kernel position
                each row is a point, each column is a parameter.
         kwargs:
-            bandwidth : The bandwidth of the kernel used for smoothing
+           bandwidth : The bandwidth of the kernel used for smoothing
             dim_names : sequence of dimension names, e.g. ('m1', 'z', 'chi_eff')
                         values must be strings
+        Example:
+        >>> #For two dimensional case
+        >>> rndgen = np.random.RandomState(seed=3575)
+        >>> mean1, sigma1 = 3.0, .25
+        >>> mean2, sigma2 = 14.0, 1.5
+        >>> n_samples = 1000
+        >>> sample1 = rndgen.normal(mean1, sigma1, size=n_samples)
+        >>> sample2 = rndgen.normal(mean2, sigma2, size=n_samples
+        >>> sample = np.vstack((sample1, sample2)).T #shape of data (n_points, n_features)
+        >>> kde = SimpleKernelDensityEstimation(sample, dim_names=['mass1', 'mass2'])
+        >>> minx, maxx = np.amin(sample[:, 0]), np.amax(sample[:, 0])
+        >>> miny, maxy = np.amin(sample[:, 1]), np.amax(sample[:, 1])
+        >>> x = np.linspace(minx, maxx, 100)
+        >>> y = np.linspace(miny, maxy, 100)
+        >>> XX, YY = np.meshgrid(x, y)
+        >>> eval_pts = np.array(list(map(np.ravel, [XX, YY]))).T
```

Comment:
This line is much too complicated to understand, please write it in a simpler way. Is it necessary to use `list(map(..))` ?  If the result of np.ravel on XX and YY is a numpy array, why would you need to make a list of the two and then convert it to an array and then take the transpose to get what you want?  Can you look for a numpy function that just converts ravel(XX) and ravel(YY) into the format you need?